op-chain-ops/genesis/config.go

```diff
@@ -920,6 +920,9 @@ type DeployConfig struct {
 
 	// DeployCeloContracts indicates whether to deploy Celo contracts.
 	DeployCeloContracts bool `json:"deployCeloContracts"`
+	// Unused, added to make strict config parsing possible
+	ProxyAdminOwnerIsMultiSig *bool           `json:"proxyAdminOwnerIsMultisig,omitempty"`
+	ExternalSuperchainConfig  *common.Address `json:"externalSuperchainConfig,omitempty"`
```

Comment:
In other deploy configs, we don't use pointer for `bool` and `common.Address`.
```suggestion
	ProxyAdminOwnerIsMultiSig bool           `json:"proxyAdminOwnerIsMultisig,omitempty"`
	ExternalSuperchainConfig  common.Address `json:"externalSuperchainConfig,omitempty"`
```

Reply:
I thought the point for this that the fields get a `nil` value, and not the default empty value.

Reply:
Oh I see, that makes sense

Reply:
No in this case this was more for the field not being required to be present in the config, which may or may not be what we want, but I think it's better to not be strict here for now.

Since we don't use the value currently we are also spared from the ugliness of dereferencing the bool somewhere.
This is one of the more annoying things in go's parser.

Reply:
Ok nevermind, I was wrong again